src/ascend.ts

```diff
@@ -0,0 +1,292 @@
+import {
+  containsText,
+  eudoraItem,
+  getCampground,
+  getWorkshed,
+  print,
+  toInt,
+  use,
+  visitUrl,
+  xpath,
+} from "kolmafia";
+import { $item, $items, $stat, get, have } from ".";
+import { ChateauMantegna } from "./resources";
+
+export enum lifestyle {
+  casual = 1,
+  softcore = 2,
+  normal = 2,
+  hardcore = 3,
+}
+
+export class Path {
```

Comment:
put all the path stuff into a Path.ts
combine the list into the class itself as static